### PR TITLE
Adding beta 4 networking section

### PR DESCRIPTION
**Pull request by calldelegation**
Adding beta 4 as the latest version of the testnets.

TODO:
- [ ] Update versioning of each compatible component when available to the team

### CODE REVIEW

File: src/networks/beta-4.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# The `beta-4` testnet
+
+The `beta-4` network is the latest Fuel testnet. It builds on the foundation of `beta-3``, enhancing public P2P access, allowing parallel predicate execution, and introducing a newly redesigned bridging approach that ensures security for full rollups.
```
Comment (sarahschwartz) [inline src/networks/beta-4.md]: "`beta-3``" should be "`beta-3`"

File: src/networks/beta-4.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Sepolia block explorer: [https://sepolia.etherscan.io/](https://sepolia.etherscan.io)
+
+🚰 Faucet - Use the faucet to get test ETH to deploy contracts with or to interact with contracts. Available here: [https://faucet-beta-4.fuel.network/](https://faucet-beta-4.fuel.network/).
+
+📃 GraphQL endpoint - The Fuel Core node uses GraphQL instead of JSON RPC. A playground for the public GraphQL endpoint for beta-4 is available at [https://beta-4.fuel.network/playground](https://beta-4.fuel.network/playground).
+
+🔍 Block explorer - A block explorer (still heavily in development) is available at [https://fuellabs.github.io/block-explorer-v2/](https://fuellabs.github.io/block-explorer-v2/). Be sure to select `beta-4` from the dropdown on the top right.
+
+Join the [Fuel Labs Discord](https://discord.com/invite/fuelnetwork) and head to the 🧪︱testnet channel to get support from our team.
```
Comment (sarahschwartz) [inline src/networks/beta-4.md]: We should link to the forum instead

File: src/networks/beta-4.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+🔍 Block explorer - A block explorer (still heavily in development) is available at [https://fuellabs.github.io/block-explorer-v2/](https://fuellabs.github.io/block-explorer-v2/). Be sure to select `beta-4` from the dropdown on the top right.
+
+Join the [Fuel Labs Forum](https://forum.fuel.network/) to get support from our team and others building on Fuel!
+
+## SDK Versioning
+<!-- TODO: UPDATE VERSIONING HERE -->
+Version 0.38.0 or higher is the recommended version of the TS SDK on `beta-4`.  
+
+<!-- TODO: UPDATE VERSIONING HERE -->
+Version 0.39.0 or higher is the recommended version for the Rust SDK on `beta-4`.
```
Comment (Voxelot) [inline src/networks/beta-4.md]: 0.46.0

File: src/networks/beta-4.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+📃 GraphQL endpoint - The Fuel Core node uses GraphQL instead of JSON RPC. A playground for the public GraphQL endpoint for beta-4 is available at [https://beta-4.fuel.network/playground](https://beta-4.fuel.network/playground).
+
+🔍 Block explorer - A block explorer (still heavily in development) is available at [https://fuellabs.github.io/block-explorer-v2/](https://fuellabs.github.io/block-explorer-v2/). Be sure to select `beta-4` from the dropdown on the top right.
+
+Join the [Fuel Labs Forum](https://forum.fuel.network/) to get support from our team and others building on Fuel!
+
+## SDK Versioning
+<!-- TODO: UPDATE VERSIONING HERE -->
+Version 0.38.0 or higher is the recommended version of the TS SDK on `beta-4`.
```
Comment (Voxelot) [inline src/networks/beta-4.md]: 0.51.0

File: src/networks/beta-3.md [hunk 5]
```diff
@@ -20,13 +20,13 @@ Goerli block explorer: [https://goerli.etherscan.io/](https://goerli.etherscan.i
...
 
-Join the [Fuel Labs Discord](https://discord.com/invite/fuelnetwork) and head to the 🧪︱testnet channel to get support from our team.
+Join the [Fuel Labs Forum](https://forum.fuel.network/) to get support from our team and others building on Fuel!
 
 ## SDK Versioning
 
 Version 0.38.0 or higher is the recommended version of the TS SDK on `beta-3`.  
 
-Version 0.39.0 or higher is the recommended version for the Rusk SDK on `beta-3`.
+Version 0.39.0 or higher is the recommended version for the Rust SDK on `beta-3`.
```
Comment (sarahschwartz) [inline src/networks/beta-3.md]: can we also specify an upper version limit here?

File: src/networks/networks.md [hunk 6]
```diff
@@ -2,12 +2,18 @@
 
 On this page, you'll find information on the different testnet networks.
 
-## `beta-3` testnet
-
+## `beta-4` testnet
 <!-- This example should include a description for the latest network -->
 <!-- latest_network:example:start -->
-The `beta-3` network is the latest Fuel testnet. It expands on the features of `beta-2`, introducing P2P networking and the ability to run synchronizing full nodes.
+The `beta-4` network is the latest Fuel testnet. It builds on the foundation of `beta-3``, enhancing public P2P access, allowing parallel predicate execution, and introducing a newly redesigned bridging approach that ensures security for full rollups.
```
Comment (sarahschwartz) [inline src/networks/networks.md]: "`beta-3``" should be "`beta-3`"

File: src/networks/beta-3.md (hunk 5 above)
Comment (calldelegation) [reply src/networks/beta-3.md]: Yes good call will stop at current version for beta-4

**Review by sarahschwartz — APPROVED** [review]
LGTM

**Pull request merged**